lib/charliecloud.py

```diff
@@ -630,6 +630,7 @@ def unpack_create_ok(self):
          rmtree(self.unpack_path)
 
    def unpack_delete(self):
+      VERBOSE("unpack path: %s" % self.unpack_path)
```

Comment:
I moved this to the beginning of the function so it's output even if the image doesn't exist.